### PR TITLE
[BEAM-2804] support TIMESTAMP in sort

### PR DESCRIPTION
**Pull request by zangshayang1**
This PR enables beam-sql clause : order by [TIMESTAMP]. A test case has been added to sdks/java/extensions/sql/src/test/java/org/apache/beam/sdk/extensions/sql/impl/rel/BeamSortRelTest.java

### CODE REVIEW

**Comment by zangshayang1** [comment]
restart test please

**Comment by zangshayang1** [comment]
retest this please

File: sdks/java/extensions/sql/src/test/java/org/apache/beam/sdk/extensions/sql/impl/rel/BeamSortRelTest.java [hunk 1]
```diff
@@ -92,6 +92,27 @@ public void testOrderBy_basic() throws Exception {
   }
 
   @Test
+  public void testOrderBy_timestamp() throws Exception {
+    String sql = "INSERT INTO SUB_ORDER_RAM(order_id, site_id, price)  SELECT "
+        + " order_id, site_id, price "
```
Comment (XuMingmin) [inline sdks/java/extensions/sql/src/test/java/org/apache/beam/sdk/extensions/sql/impl/rel/BeamSortRelTest.java]: can you add and check `order_time` in the output column list?

Comment (zangshayang1) [reply sdks/java/extensions/sql/src/test/java/org/apache/beam/sdk/extensions/sql/impl/rel/BeamSortRelTest.java]: But the output table is defined in `prepare()` and used by other test cases. A simpler way is to prepare a different output table in my test case. What do you think? @XuMingmin

**Pull request closed without merging**